rbx_dom_lua/src/EncodedValue.spec.lua

```diff
@@ -47,6 +47,36 @@ return function()
 		expect(decoded).to.equal(output)
 	end)
 
+	it("should decode a 'Faces' bit-mask value", function()
+		local input = {
+			Type = "Faces",
+			Value = 0b_111111,
+		}
+
+		local ok, decoded = EncodedValue.decode(input)
+		assert(ok, decoded)
+
+		for _,normalId in pairs(Enum.NormalId:GetEnumItems()) do
+			local set = decoded[normalId.Name]
+			expect(set).to.equal(true)
+		end
+	end)
+
+	it("should decode an 'Axes' bit-mask value", function()
+		local input = {
+			Type = "Axes",
+			Value = 0b_111,
+		}
+
+		local ok, decoded = EncodedValue.decode(input)
+		assert(ok, decoded)
+
+		for _,axis in pairs(Enum.Axis:GetEnumItems()) do
+			local set = decoded[axis.Name]
+			expect(set).to.equal(true)
+		end
```

Comment:
Can we check the axes individually here? Using a loop here when there are three items feels kinda funny.

If possible, we should also expand the set of tests a little bit here. Try all the axes, none of the axes, and maybe one axis.

Reply:
I adjusted the code for the original Axes test as requested. I also added an additional mixed bit-input test for both Axes and Faces to ensure the stability of this encoding technique.